sql/expression/like.go

```diff
@@ -110,6 +110,10 @@ func (l *Like) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 		return nil, err
 	}
 
+	if likeMatcher == nil {
```

Comment:
Where's the test for this fix?

Reply:
The one in queries.go triggered this originally but I can add more.